01-html/01-koajs.html

```diff
@@ -1,4 +1,671 @@
 <!-- http://koajs.com/
 Don't worry about all the content - it's an awfully long page.
 I picked this because it's a pretty simple layout.
- -->
\ No newline at end of file
+ -->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Koa - next generation web framework for node.js</title>
+</head>
+<body>
+  <header>
+    <h1>koa</h1>
+    <h5>next generation web framework for node.js</h5>
+    <nav>
+      <ul>
+        <button type="button">contents</button>
+        <li><a href="introduction">introduction</a></li>
+        <li><a href="application">application</a></li>
+        <li><a href="context">context</a></li>
+        <li><a href="request">request</a></li>
+        <li><a href="response">response</a></li>
+        <li><a href="links">links</a></li>
+      </ul>
+    </nav>
+  </header>
+  <hgroup>
+    <h2>Introduction</h2>
+      <p>Koa is a new web framework designed by the team behind Express, which aims to be a smaller, more expressive, and more robust foundation for web applications and APIs. Through leveraging generators Koa allows you to ditch callbacks and greatly increase error-handling. Koa does not bundle any middleware within core, and provides an elegant suite of methods that make writing servers fast and enjoyable.</p>
+    <h2>Installation</h2>
+      <p>Koa works out of the box with recent versions of Node. To use Koa with 0.11.x
+        you must use the <em>--harmony</em> or <em>--harmony-generators</em> flag.</p>
```

Comment:
`<em>` tags are typically used for 'emphasis', which means italics most of the time. You could change the style to match whatever you like, of course, but the `em` tag should probably be reserved for text you want to emphasize, not just format in a particular way.